### ISSUE TITLE
Search Page

### ISSUE THREAD

**Issue by ana675** [TheAlphamerc/flutter_twitter_clone#44]
Will  SearchPage in future show the posts of random users and also can we add 2-3 picture from once to be visible like grid in feed page?

**Comment by TheAlphamerc** [comment]
@ana675 can you explain this feature in detail?

**Comment by ana675** [comment]
@TheAlphamerc  I mean will in the some of next update "search page"  show the post of the random users. And also can users in future share 2-3 picture in same time like here? I hope that I explained..

![tw](https://user-images.githubusercontent.com/66259777/83637655-3d508f80-a5a8-11ea-8492-55111b8ad6ab.png)

**Comment by TheAlphamerc** [comment]
Yes, I will consider you suggestion in next update. Thanks for taking interest in `Fwitter`
